waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -663,12 +670,12 @@ proc resume*(ws: WakuStore, peerList: Option[seq[RemotePeerInfo]] = none(seq[Rem
   ## The history gets fetched successfully if the dialed peer has been online during the queried time window.
   ## the resume proc returns the number of retrieved messages if no error occurs, otherwise returns the error string
   
-  var currentTime = epochTime()
-  var lastSeenTime: float = findLastSeen(ws.messages.allItems())
+  var currentTime = getNanosecondTime(epochTime())
+  var lastSeenTime: int64 = findLastSeen(ws.messages.allItems())
   debug "resume", currentEpochTime=currentTime
   
   # adjust the time window with an offset of 20 seconds
-  let offset: float64 = 200000
+  let offset: int64 = 200000
```

Comment:
The time literals need to be converted to the nanosecond, in here `20000` is in millisecond (supposed to represent 20 seconds), without conversion, it is rendered as `20000` nanoseconds.

Reply:
BTW, `200000` has an issue, it is 200 seconds not 20 seconds, please drop one zero from it (it is not related to your PR, I just realized this is a bug)